nixpkgs_review/cli/pr.py

```diff
@@ -66,6 +66,7 @@ def pr_command(args: argparse.Namespace) -> str:
                     allow=allow,
                     checkout=checkout_option,
                     sandbox=args.sandbox,
+                    nix_flavor=args.nix_flavor,
```

Comment:
you can resolve nix_flavor into paths to `nix` and `nix-shell` here

Reply:
I would like to not resolve it to a path. This would keep things simple, as only `nix` or `nom` is passed and should work as well, since it is in `PATH`.